modules/load-balancer-target-group/README.md

```diff
@@ -19,7 +19,7 @@ working sample code.
 ## How do you use this module?
 
 Imagine you've deployed Couchbase and Sync Gateway using the [couchbase-cluster
-module](https://github.com/gruntwork-io/terraform-aws-couchbase/tree/master/modules/load-balancer) and a Load Balancer
+module](/modules/couchbase-cluster) and a Load Balancer
```

Comment:
Thx for the fix. However, as this module is in the Terraform Registry, all links, must, unfortunately, be absolute. Could you turn it into an absolute URL?